draft-ietf-quic-applicability.md

```diff
@@ -454,6 +454,35 @@ Some deadlocking scenarios might be resolved by cancelling affected streams with
 STOP_SENDING or RESET_STREAM.  Cancelling some streams results in the connection
 being terminated in some protocols.
 
+# Stream Limit Commitments
+
+QUIC endpoints are responsible for communicating the cumulative limit of streams
+they would allow to be opened by their peer. Initial limits are advertised
+using the initial_max_streams_bidi and initial_max_streams_uni transport
+parameters. As streams are opened and closed they are consumed and the cumulative
+total is incremented. Limits can be increased using the MAX_STREAMS frame but
+there is no mechanism to reduce limits. 
+
+An application that uses QUIC might communicate a cumulative stream limit
+but require the connection to be closed before the limit is reached. For example,
+stopping the server to perform scheduled maintenance. Immediate connection close
+(see {{connection-termination}}) causes abrupt closure of actively used streams.
+Depending on how an application uses QUIC streams, this could be undesireable
+or detrimental to behaviour or performance. An alternative to immediate close
+is to wait for a peer to consume all of the advertised stream limit. However, the
```

Comment:
```suggestion
is to not issue any new stream credits and wait for a peer to close the connection,
either because there is no additional data to send or it is forced to close when it
consumed all of the advertised stream limit. However, the
```
Getting a bit lengthly; it's just a proposal...

Reply:
how about we put your suggested text about how endpoints might close when the limit is reached, then the sentence here can say 

"An alternative to immediate close is to stop increasing the stream limit and wait for the peer to consume the remaining streams."

?

Reply:
Sounds good!